### ISSUE TITLE
G7 - Global Search bar reads 'Search' instead of placeholder text on screenreader

### ISSUE THREAD

**Issue by enichol** [ccswbs/hjckrrh#228]
The screenreader should be announcing whatever the text in the search bar is, i.e. "Search U of G"

**Comment by jpinsent** [comment]
The label should be "Search U of G" instead of just "Search"

**Comment by mmafe** [comment]
Updated the label to read out "Search University of Guelph."

**Comment by jpinsent** [comment]
Confirmed to be working with NVDA in Chrome and FireFox.
